### ISSUE TITLE
Add Readme and a code map

### ISSUE THREAD

**Issue by interdrift** [cerobe/Pal#89]
It would be great to have a readme file to see the complete picture on what you are planning to do with the project.

**Comment by cerobe** [comment]
That's right, to be honest that's the plan, but you took me by surprise as I didn't expect that project to be noticed before it reach a certain level of stability.

the code is still in it's early stage, still missing some coverage, do not even have all the major components (a decent UI, a repository...).

The main goal is to have a deck tracker, that shows what the opponent played in the game and what you played and still have in your deck. that would be the first public version.

Then I would like to add more features like, having stats of your games, (loss, win, what opponent you encounter, what type of card), some tips for decision in game (when you can kill, when murlock combo is ready,....)

I was thinking that the wiki project could present all the majors steps and then, for the implementation we could raise issue to implement with milestone. what do you think?

**Comment by interdrift** [comment]
I think a wiki is a good move forward, it should also give a clear idea about each of the milestones and what the expected features are for each of them.

**Comment by cerobe** [comment]
All right, could you add a public email to your github account? If you're interested to work on the project, we could try to define together on what you could work.

I'll add  bit more information on the wiki.
